src/components/PlantingSites/editor/Zones.tsx

```diff
@@ -94,6 +153,30 @@ export default function Zones(props: ZonesProps): JSX.Element {
     [setZones]
   );
 
+  const popupRenderer = useMemo(
+    (): MapPopupRenderer => ({
+      className: classes.tooltip,
+      render: (properties: MapSourceProperties, onClose?: () => void): JSX.Element | null => {
+        const { name, targetPlantingDensity } = properties;
+
+        const onUpdate = (nameVal: string, targetPlantingDensityVal: number) => {
+          // TODO
+          return;
+        };
+
+        return (
+          <TooltipContents
+            name={name}
+            onClose={() => onClose?.()}
```

Comment:
👀 I didn't know you could do that, nice

Reply:
wait, i thought i saw this in your PR and lifted the idea from there.. lol

Reply:
Haha oh I wish. I usually do `onClose && onClose()`